tests/test_20_use_case_topology.py

```diff
@@ -1132,3 +1132,50 @@ def test_141_changing_invalid_vlan_range(self):
         assert l2vpn_response.get("status") == "up"
         assert l2vpn_response['current_path'] != first_path
         assert new_vlan_range == first_vlan_range
+    
+    @pytest.mark.xfail(reason="It is not verified that the corresponding L2VPN is in the OXPs.")  
+    def test_142_create_l2vpn_with_vlan_range_same_items(self):
+        """
+        Use Case 14: User requests the creation of a L2VPN with VLAN Range.
+        """
+
+        l2vpn_payload = {
+            "name": "Test L2VPN creation with VLANs range",
+            "endpoints": [
+                {"port_id": "urn:sdx:port:ampath.net:Ampath3:50","vlan": "3000:3000"},
+                {"port_id": "urn:sdx:port:sax.net:Sax01:50","vlan": "3000:3000"}
+            ]
+        }
+        response = requests.post(API_URL, json=l2vpn_payload)
+        assert response.status_code == 201, response.text
+        data = response.json()
+        l2vpn_id = data.get("service_id")
+
+        time.sleep(5)
+
+        response = requests.get(API_URL)
+        assert response.status_code == 200, response.text
+        data = response.json()
+        assert len(data) == 1, str(data)
+        assert l2vpn_id  in data, str(data)
```

Comment:
@gretelliz can you please add an extra check here to validate the status of this l2VPN as "up"?